src/main/kotlin/frc/chargers/hardware/sensors/DriverCamManager.kt

```diff
@@ -0,0 +1,27 @@
+package frc.chargers.hardware.sensors
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard
+import edu.wpi.first.networktables.NetworkTable
+import edu.wpi.first.networktables.NetworkTableEntry
+import edu.wpi.first.networktables.NetworkTableInstance
+import edu.wpi.first.cameraserver.CameraServer
+
+public class DriverCamManager(
+    public val totalDriverCams: Int = 0,
+    public val defaultResWidth: Int = 720,
+    public val defaultResHeight: Int = 1280){
+    internal var driverCameras = Array(totalDriverCams){i -> CameraServer.startAutomaticCapture(i)}.toList()
+    internal val cameraSelector: NetworkTableEntry = NetworkTableInstance.getDefault().getTable("").getEntry("CameraSelection")
```

Comment:
Will the table always be `""` or should that be a constructor parameter (perhaps with default value `""`)?

Reply:
It's just how the network tables thingy works. I don't really know the specifics of it